src/services/alert-status.js

```diff
@@ -6,13 +6,16 @@ export const FETCH_STRUCTURED_DATA_ERROR =
   'There was an error fetching the Structured Metadata from server';
 export const PEAKJS_INITIALIZE_ERROR =
   'There was an error initializing the PeakJS waveform';
+export const PEAKSJS_REACHED_END_OF_FILE =
+  'Time ahead has timespans reaching the end of media file, there is no available time to insert a new timespan';
 
 /**
  * Helper function which prepares a configuration object to feed the AlertContainer component
  * @param {number} status Code for response
+ * @param {function} clearAlert A function defined in the hosting component to clear the alert object in component's state
  */
-export function configureAlert(status) {
-  let alertObj = { alertStyle: 'danger' };
+export function configureAlert(status, clearAlert) {
+  let alertObj = { alertStyle: 'danger', clearAlert: clearAlert };
```

Comment:
Another tiny ES6/7 trick... when the property name is equal to a variable of same name, you could just do 

`{ alertStyle: 'danger', clearAlert }`

Reply:
Oh cool! Did not know that. Will keep that in mind for next time :)

Reply:
@adamjarling Or should I fix it now?

Reply:
Next time is fine.